ptvsd/_vendored/pydevd/_pydevd_bundle/pydevd_comm.py

```diff
@@ -449,16 +451,11 @@ def _on_run(self):
         """ just loop and write responses """
 
         self._stop_trace()
-        get_has_timeout = sys.hexversion >= 0x02030000 # 2.3 onwards have it.
         try:
             while True:
                 try:
                     try:
-                        if get_has_timeout:
-                            cmd = self.cmdQueue.get(1, 0.1)
-                        else:
-                            time.sleep(.01)
-                            cmd = self.cmdQueue.get(0)
+                        cmd = self.cmdQueue.get(1, 0.1)
```

Comment:
Is this okay for upstream?

Reply:
It is already on upstream https://github.com/fabioz/PyDev.Debugger/blob/master/_pydevd_bundle/pydevd_comm.py#L458

@fabioz usually makes the ganges upstream then ports them over here

Reply:
Yeap... that code was for some really old versions of Python (2.3 and before), so, now that the minimum is 2.6 it should be safe.